static/app/views/replays/detail/network/index.tsx

```diff
@@ -0,0 +1,173 @@
+
+type Props = {
+  event: EventTransaction;
+  networkSpans: NetworkSpan[];
+};
+
+function NetworkList({event, networkSpans}: Props) {
+  const {startTimestamp} = event;
+  const [sortConfig, setSortConfig] = useState<ISortConfig>({
+    by: 'startTimestamp',
+    asc: true,
+  });
+
+  const handleSort = useCallback(
+    (heading: typeof sortConfig.by) => {
+      setSortConfig(prevSort => {
+        if (prevSort.by === heading) {
+          return {by: heading, asc: !prevSort.asc};
+        }
+
+        return {by: heading, asc: true};
+      });
+    },
+    [sortConfig]
+  );
+
+  const networkData = useMemo(
+    () => sortNetwork(networkSpans, sortConfig),
+    [networkSpans, sortConfig]
+  );
+
+  const columns = [
+    t('status'),
+    t('path'),
+    t('type'),
+    t('duration'),
+    <SortItem key="timestamp" onClick={() => handleSort('startTimestamp')}>
+      {t('timestamp')}{' '}
+      <IconArrow
+        color="gray300"
+        size="xs"
+        direction={sortConfig.by === 'startTimestamp' && sortConfig.asc ? 'down' : 'up'}
+      />
+    </SortItem>,
+  ];
+
+  const renderTableRow = (network: NetworkSpan, index: number) => {
+    const networkStartTimestamp = network.startTimestamp * 1000;
+    const networkEndTimestamp = network.endTimestamp * 1000;
+
+    return (
+      <Fragment key={index}>
+        <Item>{<StatusPlaceHolder height="20px" />}</Item>
+        <Item color={theme.gray400}>
+          <Text>{network.description || <Placeholder height="24px" />}</Text>
+        </Item>
+        <Item>
+          <Text>{network.op}</Text>
+        </Item>
+        <Item numeric>
+          {`${(networkEndTimestamp - networkStartTimestamp).toFixed(2)}ms`}
+        </Item>
+        <Item numeric>{showPlayerTime(networkStartTimestamp, startTimestamp)}</Item>
+      </Fragment>
+    );
+  };
+
+  return (
+    <StyledPanelTable
+      columns={columns.length}
+      isEmpty={networkData.length === 0}
+      emptyMessage={t('No related network requests found.')}
+      headers={columns}
+      disablePadding
+    >
+      {networkData.map(renderTableRow) || null}
+    </StyledPanelTable>
+  );
+}
+
+const Item = styled('div')<{color?: string; numeric?: boolean}>`
+  display: flex;
+  align-items: center;
+  max-height: 28px;
+  color: ${p => p.color || p.theme.subText};
+  border-radius: 0;
+  padding: ${space(0.75)} ${space(1.5)};
+  background-color: ${p => p.theme.background};
+  min-width: 0;
+
+  ${p => p.numeric && 'font-variant-numeric: tabular-nums;'}
+
+  /* Make odd rows have different background colors */
+  &:nth-child(10n + 5),
+  &:nth-child(10n + 4),
+  &:nth-child(10n + 3),
+  &:nth-child(10n + 2),
+  &:nth-child(10n + 1) {
+    background-color: ${p => p.theme.backgroundSecondary};
+  }
+`;
+
+const StyledPanelTable = styled(PanelTable)<{columns: number}>`
+  grid-template-columns: max-content minmax(200px, 1fr) repeat(3, max-content);
+  font-size: ${p => p.theme.fontSizeSmall};
+  line-height: 16px;
+
```

Comment:
Can we make the table internally scrollable?

I think this combo of things will do it, but it's untested (esp the selector):

```suggestion
  margin-bottom: 0;
  height: 100%
  overflow: auto;
  
  /* Make the header row sticky */
  > :nth-child(-n + ${p => p.columns}) {
    position: sticky;
    top: 0;
  }
```